README.md

```diff
@@ -235,73 +239,77 @@ Returns the time between `<dateA>` and `<dateB>` relative to `<dateB>` in a way
+| `<dateB>`   | Any value(s) [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...) |
+| `precision` | An optional [unit of measurement](https://momentjs.com/docs/#/displaying/difference), defaults to `'milliseconds'`                           |
+| `float`     | An optional `Boolean` to get the floating point result, rather than the integer value                                                        |
 
 Returns the difference in `precision` units between `<dateA>` and `<dateB>` with floating point accuracy if `float` is `true`. See [`momentjs#diff`](https://momentjs.com/docs/#/displaying/difference/).
 
 **Examples**
 
 ```hbs
-{{moment-diff '2018-01-25' '2018-01-26'}} {{!-- 86400000 --}}
-{{moment-diff '2018-01-25' '2018-01-26' precision='years' float=true}} {{!-- 0.0026881720430107525 --}}
+{{moment-diff '2018-01-25' '2018-01-26'}}
+{{! 86400000 }}
+{{moment-diff '2018-01-25' '2018-01-26' precision='years' float=true}}
+{{! 0.0026881720430107525 }}
 ```
 
 ### moment-add
+
 ```hbs
 {{moment-add <date> <number> [precision='milliseconds']}}
 ```
 
-| Parameters | Values |
-| ---------- | ------ |
-| `<date>` | An optional value [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...). Defaults to value of `moment()` |
-| `<number>` | Any value(s) [interpretable as a duration](https://momentjs.com/docs/#/durations/creating) by `moment` that is the amount of the `precision` you want to `add` to the `date` provided |
-| `precision` | An optional [unit of measurement](https://momentjs.com/docs/#/displaying/difference), defaults to `'milliseconds'` |
+| Parameters  | Values                                                                                                                                                                                |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `<date>`    | An optional value [interpretable as a date/time](https://momentjs.com/docs/#/parsing/) by `moment` (a date `String` or a `Moment` or a `Date`...). Defaults to value of `moment()`    |
+| `<number>`  | Any value(s) [interpretable as a duration](https://momentjs.com/docs/#/durations/creating) by `moment` that is the amount of the `precision` you want to `add` to the `date` provided |
+| `precision` | An optional [unit of measurement](https://momentjs.com/docs/#/displaying/difference), defaults to `'milliseconds'`                                                                    |
 
 Mutates the original moment by adding time. See [`momentjs#add`](https://momentjs.com/docs/#/manipulating/add/).
 
 **Examples**
 
 ```hbs
-{{!-- Add 6 days to a date --}}
+{{! Add 6 days to a date }}
```

Comment:
why these changes?

there are a lot of formatting changes in this PR that kinda distract from what's actually changing, ya know?